src/containers/WalletTransfer/WalletTransfer.vue

```diff
@@ -909,7 +944,104 @@ export default {
         this.toEthAddress = await this.calculateEthAddress()
       }
     },
+    getReturnObject() {
+      return this.isBitMode
+    },
+    async listenInput(input) {
+      if (input == null) {
+        return
+      }
+      this.contactSelected = input
+
+      // check tail of .bit begin
+      this.multipleAddress = []
+      this.bitError = ''
+      this.toAddress = ''
+      this.toEthAddress = ''
+      const tail = DOT_STRING + BIT
```

Comment:
Seems like `this.bitTail` and `tail` here are the same things.

Reply:
done